pytd/pandas_td/__init__.py

```diff
@@ -345,8 +345,9 @@ def to_td(frame, name, con, if_exists='fail', time_col=None, time_index=None, in
     frame = _convert_index_column(frame, index, index_label)
     frame = _convert_date_format(frame, date_format)
 
-    writer = SparkWriter(con.apikey, con.endpoint)
-    writer.write_dataframe(frame, con.database, name, mode)
+    if con.writer is None:
```

Comment:
Good catch - we can use `con` for writing data. 

Can it be simplified as follows?

```py
con.load_table_from_dataframe(frame, name, mode)
```

Reply:
Ah, that's smarter than what I wrote 80f36ae